tsconfig.json

```diff
@@ -16,7 +16,7 @@
     "forceConsistentCasingInFileNames": true,
     "types": ["vitest/globals"],
     "paths": {
-      "tinypool": ["./dist/index.d.ts"]
+      "tinypool": ["./dist/index.js"]
```

Comment:
Pointing a path namespace to a `.d.ts` files fail with Bun with following error. And actually we should point to a JS entry file and types will be loaded automatically with matching file name. 

```
1 | (function (entry, fetcher)
              ^
SyntaxError: Missing 'default' export in module '/Users/nazar/Hub/Lodestar/Projects/tinypool/dist/index.d.ts'.
      at requestImportModule (1:11)

error: script "test" exited with code 1
```